lib/tasks/generateResourcesJson.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+
+// const log = require("@ui5/logger").getLogger("builder:tasks:generateResourcesJson");
+const resourceListCreator = require("../processors/resourceListCreator");
+
+const DEFAULT_EXCLUDES = [
+	"!**/.DS_Store",
+	"!**/.eslintrc",
+	"!**/.eslintignore",
+	"!**/.gitignore"
+];
```

Comment:
Do we have a common place for such excludes?

Reply:
Not yet. Although I would not expect the last three in the `webapp` or `src` directories. Do we know of such cases?

Would it be possible to rather have an include on the expected file extensions?

Reply:
Not yet. Although I would not expect the last three in the `webapp` or `src` directories. Do we know of such cases?

Would it be possible to rather have an include on the expected file extensions?

Reply:
the file extensions can be anything required by the application, from image to web resource, therefore it is hard to define a whitelist.